.github/workflows/pr.yml

```diff
@@ -0,0 +1,44 @@
+name: Build
+
+on:
+  pull_request:
+
+jobs:
+  nightly:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+
+      - name: Install dependencies
+        run: |
+          yarn install --frozen-lockfile
+
+      - name: Update package.json
+        run: |
+          set -e
+          setSegmentKey="setpath([\"segmentKey\"]; \"${{ secrets.ANALITYCS_KEY }}\")"
```

Comment:
we should not set keys from secrets, it should be used only for (pre)-releases

@filiptronicek could you remove it?

Reply:
> we should not set keys from secrets, it should be used only for (pre)-releases

Wondered whether that was a good idea, removed it now, thanks for the input!

@akosyakov are you opposed to having artifacts in the action's outputs? I would find it very helpful so that a) as the PR creator I don't have to build the extension and upload to the PR's description and b) if I want to review a PR, I can always get the latest artifact without having to build myself, which eliminates me having to pull the latest changes, installing dependencies and packaging up the extension.

Agreed that having `yarn build` is very much enough for just CI, but to help the whole process these packaged artifacts would be cool 😄.

Reply:
We no longer upload vsix for this extension, we did it before because it was inside vscode repo, we still do this for gitpod-remote only.
If someone wants to review then he needs to clone the repo and run locally, also for testing telemetry and check them in segment by default it queues events so usually you want to change that behavior manually in the code

Reply:
Thanks for clearing this up, @jeanp413 🙏. Removed this in https://github.com/gitpod-io/gitpod-vscode-desktop/pull/22/commits/d80cdb4eefd635a20de049a4f9b92e2f15002892.